test/lib/mayaUsd/utils/materialx_test_data/topology_tests.mtlx

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0"?>
+<materialx version="1.38" colorspace="lin_rec709">
+
+  <!-- Simple use of channels on output node -->
+  <nodegraph name="Ng1">
+    <constant name="constant" type="color3" nodedef="ND_constant_color3">
+      <input name="value" type="color3" value="0.263273, 0.263273, 0.263273" />
+    </constant>
+    <output name="float_output" type="float" nodename="constant" channels="g" />
+  </nodegraph>
+  <standard_surface name="Surf1" type="surfaceshader">
+    <input name="base" type="float" nodegraph="Ng1" output="float_output" />
+  </standard_surface>
+  <surfacematerial name="Channel1" type="material" topo="Channel1_topo.mtlx">
+    <input name="surfaceshader" type="surfaceshader" nodename="Surf1" />
+  </surfacematerial>
```

Comment:
These are all the test cases, the "topo" attribute on the "surfacematerial" node indicates which file contains the output we seek. As you can see quite a few of these graphs will end up with the same baseline topo result.